packages/devtools_app/lib/src/screens/inspector/inspector_tree_controller.dart

```diff
@@ -973,6 +976,18 @@ class _InspectorTreeState extends State<InspectorTree>
   Widget build(BuildContext context) {
     super.build(context);
     final treeControllerLocal = treeController;
+
+    ga.select(
+      gac.inspector,
+      'InspectorTreeBuild',
+      nonInteraction: true,
+      screenMetricsProvider: () => InspectorTreeBuildMetrics(
+        isSummaryTree: widget.isSummaryTree,
+        buildCount: ++_buildCount,
+        rowCount: treeControllerLocal?.numRows,
+      ),
+    );
+
```

Comment:
I wouldn't put this in build. We aren't concerned with counting builds from the widget lifecycle as much as we are concerned with when the inspector tree data is updated and if the data is showing 0 rows then we know we have a problem.

Another thing to think about. Analytics should be tracking something that is stable over time - if code in the inspector changes in such a way that we are intentionally triggering more rebuilds, when the analytics data becomes bogus since the baseline is always changing.